docs/examples/basics/demo_magneticbottle.jl

```diff
@@ -1,28 +1,34 @@
-# Tracing charged particle in the magnetic bottle.
-# This example shows how to trace protons in a stationary magnetic field that
-# corresponds to the magnetic bottle.
+# ---
+# title: Electron motion in a magnetic bottle
+# id: demo_proton_bottle
+# date: 2023-04-20
+# author: "[Hongyang Zhou](https://github.com/henry2004y)"
+# julia: 1.9.0
+# description: Tracing charged particle in a magnetic bottle
+# ---
+
+# This example shows how to trace non-relativistic and relativistic electrons in a
+# stationary magnetic field that corresponds to a magnetic bottle.
 # Reference: https://en.wikipedia.org/wiki/Magnetic_mirror#Magnetic_bottles
-#
-# Hongyang Zhou, hyzhou@umich.edu
 
```

Comment:
Why aren't images exported here?

Reply:
Ah, because I forgot to add the JSServe Page settings here. Will be fixed.